packages/demand-api/src/quotes/types.ts

```diff
@@ -1,11 +1,18 @@
 import { VehicleAttributes } from '../sharedTypes'
+import { HttpResponse } from '../http/types'
 
 export type QuotesAvailabilityParams = {
   originPlaceId: string
   destinationPlaceId?: string
   dateRequired?: string
 }
 
+export type QuotesSearchParams = {
+  originPlaceId: string
+  destinationPlaceId: string
```

Comment:
Could we is such way rename the rest parameters and make them as on SDK [schema](https://developer.karhoo.com/reference#quotes-object-schemas) ?

Reply:
Good point 👍 but I tried to keep it similar to checkAvailability params. Maybe we need to discuss it all together and agree on correct naming later